docs/installation.rst

```diff
@@ -0,0 +1,236 @@
+.. _installing_ansible_navigator:
+
+Installing ansible-navigator with execution environment support
+###############################################################
+
+.. contents::
+   :local:
+
+Linux
+*****
+
+Requirements
+============
+
+* Either ``podman`` or ``Docker for Linux``
+* Internet access (during initial installation)
+
+Install the desired container engine for execution environment support
+======================================================================
+
+* Follow the ``podman`` installation instructions for the appropriate `Linux distribution <https://podman.io/getting-started/installation>`__.
+* Follow the ``Docker for Linux`` installation instructions for the appropriate `Linux distribution <https://docs.docker.com/engine/install/>`__.
+
+Install ansible-navigator
+=============================
+
+#. Install the python package manager using the system package installer (e.g.):
+
+   .. code-block:: console
+
+    sudo dnf install python3-pip
+
+#. Install ansible-navigator:
+
+   .. code-block:: console
+
+    python3 -m pip install ansible-navigator --user
+
+#. Add the installation path to the PATH (e.g.):
+
+   .. code-block:: console
+
+    echo 'export PATH=$HOME/.local/bin:$PATH' >> ~/.zshrc
```

Comment:
Defaulting to zsh instructions here (instead of bash used by default on RH distros) seems weird to me.

Is there a file (`.profile` or something?) that gets sourced by default from all the common shells so long as you promise to write sh-compatible code in it? :sweat_smile: I'm not sure if that exists. I'm not sure what the best thing to do here is. Maybe a docs team question.

Reply:
It's `.zprofile` for zsh and `.profile` for some others. But it's usually sourced at session respawn, not on each terminal shell invocation.

Reply:
.zprofile doesn't solve the problem because it's still zsh specific. I wish there were some generic file that gets sourced by the sh compatible shells, as long as you promise to remain sh compatible, but I guess not.

Yeah, I'm not sure what the best solution is here.